app/views/course/assessment/questions/_form.html.slim

```diff
@@ -2,4 +2,5 @@
 = f.input :description
 = f.input :staff_only_comments
 = f.input :maximum_grade
-= f.association :skills, as: :token, collection: current_course.assessment_skills, multiple: true
+= f.input :skill_ids, as: :token, collection: current_course.assessment_skills,
+  multiple: true, label: 'Skills'
```

Comment:
Missing translation here

Reply:
Ok, will add translation for this field. Actually, all the files using ```simple_form``` without providing the label can be considered as missing translation. But I think I will leave them for now.

Reply:
OK, thanks!